src/en/docs/usage-basic-overview.md

```diff
@@ -5,7 +5,24 @@ priority: 2
 translationKey: usage-basic-overview
 ---
 
-> TODO: Explain the basics of using Photoview:
->
-> - How the timeline works, and how to navigate to individual albums from the timeline.
-> - How the albums section works, and how nested albums works.
+{% optimizedImage './timeline.png', 'Screenshot of Photoview', 'class="block w-4/5 mx-auto my-6"' %}
+
+## Timeline view ##
+After loggin in to Photoview, you will see all of your photos in reverse time order with the newest ones first and the oldest ones at the bottom. The photos are grouped by the day they were taken. This is called the _timeline view_ and you can always return to this view by clicking the top-most icon on the panel at the left of the screen.
+At first it seems that there are only a few rows of photos when you scroll down, but Photoview will fill the webpage with more and more photos when you continue to scroll down.
+
+In timeline view you can do the following:
+- **Click on a photo** to display a large version of that photo. From there you can display the next photo (moving _down_ in the timeline) by clicking on the right arrow, or display the previous photo (moving _up_ in the timeline) by clicking on the left arrow. 
+There is an "X" in the top left of the photo. Clicking this takes you back to the timeline view.
+- **Click on the heart.** Placing the mouse cursor over the photo, you will see two symbols: In the bottom-left of the photo is a outline of a ♥ and in the upper-right there is an &#9432;. Clicking the heart-outline changes it to a solid heart to indicate that the photo is one of your favourites. 
+On the top of the screen there is a `checkbox` with the text "Show only favourites" next to it and checking it will only show the photos that you have marked with the heart.
+- **Click on the &#9432;** to open up a panel on the right side of the screen where you can see some metadata for that photo, such as when the photo was taken, with which camera, its exposure settings etc. It also shows the _path to the album/folder_ where the photo is stored. Clicking on the album path allows you to see all the photos from that album. Furthermore you can also see which faces that have been identified and the location where it was taken (if known). At the bottom of the panel, you have the option to download this photo to your computer or create a link that can be shared with others. More on that later.
```

Comment:
> which faces that have been identified

... (if this feature is enabled)

Reply:
Did you decide to not implement this change or just missed to add it?

Reply:
You didn't answer me in this thread, so I'm just checking) This is not critical, just want to make sure you didn't miss it)